web/files/files.go

```diff
@@ -251,21 +294,27 @@ func ReadFileHandler(c *gin.Context) {
 
 	// Path /files/download is handled specifically to download file
 	// form their path
+	var doc *vfs.FileDoc
+	var disposition string
 	if fileID == "download" {
-		pth := c.Query("path")
-		err = vfs.ServeFileContentByPath(vfsC, pth, c.Request, c.Writer)
+		disposition = "attachment"
```

Comment:
I'm not sure it's a good idea to use an attachment disposition here. `/files/download` will be used by the clientside apps to load their JS, CSS and images, and I don't know how the browsers handle the attachment disposition for these cases. We can start with that and see later if it causes some issues.